src/main/resources/application.conf

```diff
@@ -2,8 +2,8 @@ stripe-scala {
   endpoint="https://api.stripe.com"
   endpoint=${?STRIPE_ENDPOINT}
 
-  file-upload-endpoint="https://uploads.stripe.com"
-  file-upload-endpoint=${?STRIPE_FILE_UPLOAD_ENDPOINT}
+  fileUploadEndpoint="https://uploads.stripe.com"
```

Comment:
Okay my bad, had no idea there were conventions. I will change the config to reflect this